heartbeat.py

```diff
@@ -42,29 +46,33 @@ def expand_message(self, message):
         return message
 
     def do_pass(self):
-        if self.get('state') != 'passing':
+        now = int(time.time())
```

Comment:
With the config fields changing to a timestamp, I think `expand_message` needs to invoke `format` so that the e.g. last pass time is conveyed as a human readable string in the message. Or did I miss something?

Reply:
you are right, sorry I didn't check the implementation thoroughly, I will follow the original implementation